File: Exercicios_guanaraba/Com_Dart/bin/ex024_verificando_a_primeira_palavra.dart
/*

Crie um programa que leia o nome de uma cidade e diga se ela começa ou não com o nome 
'Santo'.

*/

import 'dart:io';

void main(){


  var palavra = "santo";
  print('Insira o nome completo: ');
  var user = stdin.readLineSync();
  var novonome = user.trim().split(' ')[0].toLowerCase();
  if (novonome == palavra){
    print('O primeiro nome digitado correponde com $palavra');
  }else{
    print('O primeiro nome digitado não corresponde com $palavra');
  }





}

File: Exercicios_guanaraba/Com_Ruby/exercicios/ex024_Verificando_a_primeira_palavra.rb
=begin

Crie um programa que leia o nome de uma cidade e diga se ela começa ou não com o nome 
'Santo'.

=end

palavra = 'Santo'

print 'Digite o nome completo'
user = gets.chomp

#A função split do ruby remove os espaços em branco
novonome = user.split(" ")
puts novonome[0]

if novonome[0].upcase == palavra.upcase
    puts "O nome digitado começa com #{palavra}"
else
    puts "O nome digitado não começa com #{palavra}"
end